cmd/skaffold/app/cmd/flags.go

```diff
@@ -543,6 +543,14 @@ var flagRegistry = []Flag{
 		FlagAddMethod: "DurationVar",
 		DefinedOn:     []string{"deploy", "dev", "run", "debug"},
 	},
+	{
+		Name:          "verify-env-file",
```

Comment:
Does it make sense to have a more generic name for this? Maybe `--env-file`? Just thinking maybe is better to have a single name for verify and custom actions (https://github.com/GoogleContainerTools/skaffold/issues/8560)

Reply:
Good point!  Done